drivers.xml

```diff
@@ -53,6 +53,10 @@
             <driver name="AstroPhysics">indi_lx200ap</driver>
             <version>1.0</version>
         </device>
+        <device label="AstroPhysics" manufacturer="Astrophysics">
```

Comment:
Label should be Astrophysics Testing, not just Astrophysics. Label is the final driver name.

Reply:
Done.